Finage datafeeds and beacons

Ashar2shahid opened the pull request:
This adds the following: 

**FOREX** 
MXN
EUR
GBP
AUD
CNH
JPY
NZD

**STOCK**
TSLA
AAPL
AMZN
META
SNAP
AXP (AMERICAN EXPRESS)
CSCO (CISCO SYSTEMS)
AT&T (T)
DIS (WALT DISNEY CO)
ABBV (AbbVie)
MMM (3M)
JPM (JPMORGAN CHASE & CO)
JNJ (JOHNSON & JOHNSON)

All pairs are in USD

Beacons are currently not added to the explorer, @aquarat I guess I should wait for #95 before adding those fields in

aquarat commented:
Hey Ashar, this is a great start, thanks.
These will just need the "Forex" removed from the names (can remain in the descriptions).
We'll also need to add explorer/market metadata, like pricingCoverage and prefix symbol (like `$`).

aquarat commented:
I marked this PR as ready to review so I can see a preview build :+1:

Ashar2shahid commented:
I've added the deployment files for today'd deployment which I will forward to Cam. I've also removed deployment files for gcp because the deployment was unsuccessful. Once Cam confirms the deployment we can verify and get this merged

aquarat commented:
Are you going to fix this soon?

Ashar2shahid commented:
> Are you going to fix this soon?

are you referring to the CI/CD ?

Ashar2shahid commented:
Fixed the CI/CD issues. We should be able to merge this now. The AWS deployment is confirmed and we can create a separate PR for the GCP deployment

aquarat commented:
> Fixed the CI/CD issues. We should be able to merge this now if the AWS is confirmed and we can create a separate PR for the GCP deployment

Awesome, thanks for fixing it :muscle:

aquarat commented:
Can someone message me on Keybase with the monitoring credentials for this PR please :)
@Cam maybe or @Ashar2shahid

bbenligiray commented:
I handled the deployment files and shared them and the receipt with Ashar

aquarat commented:
We're definitely getting the `220719-0949-aws-finage` deployment's heartbeat.
We're also getting `220705-1613-gcp-finage`.

aquarat commented:
The preview looks good. I'll redeploy Telemetry+Airseeker for this ASAP.

Ashar2shahid commented:
Cool, we should have this merged as soon as you deploy

Ashar2shahid commented:
@aquarat can you share the status of airseeker? were you able to use the `220719-0949-aws-finage` deployment?

bbenligiray commented:
@Ashar2shahid We're waiting to hear back from Finage about that deployment

umersin61 commented:
@Ashar2shahid please add the AAPL Beacon.

@bbenligiray please don't redeploy airseeker until the above is done & this [PR](https://github.com/api3dao/api3-explorer/pull/168) has been merged.

bbenligiray reviewed (approved):
Airseeker redeployed

bbenligiray commented:
Merging after the market update
